cmd/manager/main.go

```diff
@@ -68,11 +76,11 @@ func main() {
 			os.Exit(1)
 		}
 		var major, minor int
-		if major, err = strconv.Atoi(serverVersion.Major); err != nil {
+		if major, err = strconv.Atoi(getCleanVersionserverVersion.Major); err != nil {
```

Comment:
if we use the `semver` parser has suggested in previous comment, we can simplify this code  with something like
```golang
minServerVersion, err := semver.Make("1.10.0")
curServerVersion, err := semver.Make(serverVersion.String())
if curServerVersion.Compare(minServerVersion) < 0 {
    if err = os.Setenv(kanaryConfig.KanaryStatusSubresourceDisabledEnvVar, "1"); err != nil {
        log.Error(err, "")
	os.Exit(1)
    }
}}

```